packages/flutter/test/widgets/autocomplete_test.dart

```diff
@@ -45,446 +45,513 @@ void main() {
     User(name: 'Charlie', email: 'charlie123@gmail.com'),
   ];
 
-  group('RawAutocomplete', () {
```

Comment:
Sorry for the ugly diff in this file.  All I did was remove this group, since it was just one big group with all tests in it, and outdented the tests.  Then I added the last test in this file, "can create a field outside of fieldViewBuilder". Maybe look at the raw file.